packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListLoading.test.scss

```diff
@@ -0,0 +1,8 @@
+@import './InfiniteListTestCommon';
+
+:local {
+  .content-wrapper {
+    @include content-common(100%, #f0f8ff);
```

Comment:
lets drop the @include syntax. Lets directly add the styles.

Reply:
Done